integration_test/scripts/run-import-adapter-gcs-backup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# ADAPTER_PORT VERSION PROJECT LOCATION DATASET STORE_NAME BUCKET
+
+export PATH=$PATH:/opt/gradle/gradle-6.7/bin &&
+cd /workspace/import &&
+gradle run \
+ -Dorg.dcm4che3.imageio.codec.ImageReaderFactory=com/google/cloud/healthcare/imaging/dicomadapter/transcoder/ImageReaderFactory.properties \
+ -Dorg.dcm4che3.imageio.codec.ImageWriterFactory=com/google/cloud/healthcare/imaging/dicomadapter/transcoder/ImageWriterFactory.properties \
+ -Dexec.args="--dimse_aet=IMPORTADAPTER --dimse_port=${1} --dicomweb_address=https://healthcare.googleapis.com/${2}/projects/${3}/locations/${4}/datasets/${5}/dicomStores/${6}/dicomWeb --persistent_file_storage_location=gs://${7}/backup --oauth_scopes=https://www.googleapis.com/auth/cloud-platform --gcs_backup_project_id=${3} --persistent_file_upload_retry_amount=2 --verbose" &
+ trap "exit 0" SIGTERM; while true; do sleep 1; done
```

Comment:
Not sure if we need the trap here given we are just running this async anyway.  I believe we could remove the trap command in each of these scripts, and not bother running this script with "&" at the end

Reply:
We need only &, therefore, without this, the process starts, but does not proceed to the next step in the script, and we only see the logs that the process has started until the test stops by timeout

Reply:
We're waiting for adapter port so should be fine.